docs/photoniq/prerendering/best-practices-prerendering.md

```diff
@@ -5,26 +5,22 @@ title: Prerendering Best Practices
 
 Prerendering is an optimization technique that generates static HTML pages from dynamic content to improve website performance. The following best practices help you get the most out of Prerendering.
 
-## Selective Prerendering
-
-To maximize the benefits of prerendering, target specific areas of your website that will gain the most from this optimization technique. Typically, these are high-traffic pages or sections featuring dynamic content that seldom changes. By focusing on these areas, you ensure efficient use of resources and faster load times for your users.
-
 ## Domain Configuration
```

Comment:
Here I could mention Origin instead of Domain

Reply:
Do you mean I should replace "domain" with "origin" in this paragraph?

Reply:
Yeah, I think we should talk about origin, not domain.
So the title would be `Origin Configuration`.

Reply:
Great stuff! Updated.